apps/backend/src/config/index.ts

```diff
@@ -1,6 +1,9 @@
 import 'reflect-metadata';
 
 switch (process.env.DEPENDENCY_CONFIG) {
+  case 'base':
+    require('./dependencyConfigBase');
+    break;
```

Comment:
Why don't we use base config by default if no value is given?

Reply:
Maybe this is a good point. I will do the changes and discuss this additionally with STFC if they are OK with this way of having the base config as default.